src/main/java/ch/njol/skript/expressions/ExprClicked.java

```diff
@@ -72,7 +73,8 @@
 		SLOT(2, Slot.class, "clicked slot", "clicked slot"),
 		INVENTORY(3, Inventory.class, "clicked inventory", "clicked inventory"),
 		TYPE(4, ClickType.class, "click type", "click (type|action)"),
-		ACTION(5, InventoryAction.class, "inventory action", "inventory action");
+		ACTION(5, InventoryAction.class, "inventory action", "inventory action"),
+		ENCHANT_BUTTON(6, Number.class, "clicked enchantment button", "(clicked [enchant[ment]] button|enchant[ment] button clicked)");
```

Comment:
`enchantment button clicked` doesn't really make sense, does it?

Reply:
Since it only returns a number, probably not?
`if the enchantment button clicked is 1` sounds okay to me, although it is a bit odd.